lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -468,6 +468,21 @@ class DecomposeAtenLogSoftmaxIntOp
 };
 } // namespace
 
+// Decompose aten.log_softmax op into: log(softmax(x))
```

Comment:
Nit: `log_softmax` -> `_log_softmax`, and `softmax(x)` -> `_softmax(x)`, so that's it's clear why this is different to regular `log_softmax`